client/react/component/ClientServices.js

```diff
@@ -0,0 +1,358 @@
+    { id: '398b3646-3468-400c-9dac-36dfda193847', text: 'HEALing Communities Health Center' },
+    { id: '4dbaabac-bf4f-4988-bb14-7e659bb7978c', text: 'Chris180 at Promise Center' },
+    { id: '6f35ea09-50cf-4cfc-a973-8222384bb09d', text: 'My Sisters House' },
+    { id: '7df76b4a-b3dd-42b9-9069-859f3e8ccbe4', text: 'ProjectName_1459515376900' },
+    { id: '1b90b45f-3cfa-45a4-b775-a32bcfd56fd3', text: 'ProjectName_1459515376900' },
+    { id: '17bc369f-c9be-4ec5-8471-0c55ade8efc7', text: 'test project' },
+    { id: '1ec1fb47-6122-4f61-a596-bf7e1f75f921', text: 'ProjectName_1459515376900' },
+    { id: '558e05a6-6cd4-4ed6-8b59-b911c360d1a3', text: 'ProjectName_1459515376900' },
+    { id: '486f72c0-26a2-404f-885e-de60c6e1b0fb', text: 'test project' },
+    { id: 'db8c7696-a063-48cc-9227-3f3df3ee3c04', text: 'ProjectName_1459515376900' },
+    { id: '066c95d3-c97c-40b9-b8c1-bf12cafdfca4', text: 'ProjectName_1459515376900' },
+    { id: 'd46f8b44-3a4a-4266-8de8-eae2ed0c1563', text: 'ProjectName_1459515376900' },
+    { id: '53da6103-0155-4ebf-8452-13efe5d48209', text: 'ProjectName_1459515376900' },
+    { id: '84807d4a-a1f8-4816-9010-7590e147d6d7', text: 'ProjectName_1459515376900' },
+    { id: '53b8c2d7-e17f-4444-a720-24770ea6b3a6', text: 'ProjectName_1459515376900' },
+    { id: '9f29f656-6307-43da-93c6-256906aa5ae0', text: 'ProjectName_1459515376900' },
+    { id: 'e8d4365a-34f1-49f5-8f86-e6ec3a415a1d', text: 'test_1234' },
+    { id: '8f2b38f9-fb2b-46d6-a850-4586bd20ff86', text: 'HUD project 1' },
+    { id: '40b8ee8d-402b-483e-b8cf-007925cb2559', text: 'HUD project 1' },
+    { id: 'f324bd14-7d2b-4c74-95b6-cb4d456ad2f5', text: 'HUD project 1' },
+    { id: 'b8adfc49-078e-40ec-a130-cb705c99d65b', text: 'New project 2018' },
+    { id: 'b7c34c34-c848-46d4-9cf4-620c005c4922', text: 'test 2017' },
+    { id: 'ad25d6ab-7bb5-4304-bb93-0d345f625557', text: 'RRH 2' },
+    { id: 'eee67008-fab2-4255-9b35-6847182d0187', text: 'example1' },
+    { id: 'e6acee72-1942-41e9-8cd2-72f1bb24c615', text: 'ProjectName_1459515376900' },
+    { id: '9eea0641-65a3-4158-a4a2-06c21717daa0', text: 'ProjectName_1459515376900' },
+  ];
+
+class ClientServices extends React.Component {
+	state = {
+		project: '',
+		service: '',
+		date: '',
+		qty: '',
+		cost: '',
+		description: ''
+	}
+  constructor(props) {
+    super(props);
+
+    // This binding is necessary to make `this` work in the callback
+    this.addservices = this.addservices.bind(this);
+    this.handleProjectChange = this.handleProjectChange.bind(this);
+    this.handleServiceChange = this.handleServiceChange.bind(this);
+    this.handleQtyChange = this.handleQtyChange.bind(this);
+    this.handleDateChange = this.handleDateChange.bind(this);
+    this.handleCostChange = this.handleCostChange.bind(this);
+    this.handleDescriptionChange = this.handleDescriptionChange.bind(this);
+    this.getServices = this.getServices.bind(this);
+  }
+
+  handleProjectChange(e) {
```

Comment:
can you please unify all the below functions to take 2 arguments: key and value.
e.g.:
```
	handleInputChange(type, e) {
	  this.setState({[type]: e.target.value});
	}
```